dicom/config_service_object_stores.go

```diff
@@ -0,0 +1,117 @@
+package dicom
+
+import (
+	"bytes"
+	"encoding/json"
+	"io"
+	"net/http"
+)
+
+// CredsServiceAccess
+type CredsServiceAccess struct {
+	ProductKey     string `json:"productKey"`
```

Comment:
I am missing the endPoint param

Reply:
Good catch, added in eda33dd together with `ServiceAccount.Name` which was also missing